job-dsls/jobs/communityRelease_pipeline.groovy

```diff
@@ -10,7 +10,7 @@ def commitMsg="Upgraded version to "
 def javadk=Constants.JDK_VERSION
 def mvnVersion="kie-maven-3.5.2"
 def binariesNR=1
-String EAP7_DOWNLOAD_URL = "http://download.devel.redhat.com/released/JBoss-middleware/eap7/7.2.0/jboss-eap-7.2.0.zip"
+String EAP7_DOWNLOAD_URL = "http://download.devel.redhat.com/released/JBoss-middleware/eap7/7.3.0/jboss-eap-7.3.0.zip"
```

Comment:
I don't think this will work until we merge all PRs you opened for EAP alignment, but we can try